docs/SRS/SRS.tex

```diff
@@ -640,72 +640,72 @@ \subsection{Functional Requirements}
 
 \medskip
 \noindent \textbf{Requirement ID:} FR6\\
 \textbf{Use Case:} 5, 7\\
-\textbf{Description:} The system must provide options for users to input their price range and budgeting preferences.\\
-\textbf{Rationale:} This requirement enables users to set their budgeting preferences.\\
-\textbf{Fit Criterion:} Users should be able to set their budgeting preferences, and the system should store this information.\\
+\textbf{Description:} The system must offer a feature where users can specify their preferred price ranges for goods and set comprehensive budgeting parameters.\\
+\textbf{Rationale:} Providing users with the ability to define their price ranges and budgeting preferences is necessary for personalized shopping experience to match user constraints.\\
+\textbf{Fit Criterion:} The system must reflect user's preferences in filtering and recommending products with an accuracy rate of at least 90\%.\\
 
 \medskip
 \noindent \textbf{Requirement ID:} FR7\\
 \textbf{Use Case:} 8, 9\\
-\textbf{Description:} The system must check each product found on a user-submitted receipt and update the database with relevant purchase information.\\
-\textbf{Rationale:} This requirement ensures that the application accurately records and maintains the user's spending history, which is essential for budgeting and spending analysis.\\
-\textbf{Fit Criterion:} The system should successfully update the database with purchase information, including date, time, location, and purchase price for each product on the receipt.\\
+\textbf{Description:} The system must process each item listed on a receipt by identifying each item details. It must update the user's spending log in the database.\\
+\textbf{Rationale:} Accurately capturing detailed purchase information from receipts is important for informing users with a comprehensive overview of their grocery spending habits.\\
+\textbf{Fit Criterion:} The system should accurately extract and parse grocery item details from receipts, achieving a data accuracy of at least 90\%.\\
 
 \medskip
 \noindent \textbf{Requirement ID:} FR8\\
 \textbf{Use Case:} 11, 12\\
-\textbf{Description:} The system must analyze the user's purchase history and compare it to the budgeting preferences the user has submitted. It should suggest items to stop purchasing or recommend cheaper grocery alternatives when the user exceeds their budget.\\
-\textbf{Rationale:} This requirement supports the application's goal of helping users manage their expenses by providing personalized budgeting suggestions.\\
-\textbf{Fit Criterion:} The system should provide relevant budgeting suggestions based on the user's purchase history and budgeting preferences.\\
+\textbf{Description:} The system must analyze the user's purchase history and compare it to the budgeting preferences the user has submitted. It must also alert or recommend cheaper grocery alternatives when the user exceeds their budget.\\
```

Comment:
"It must suggestion actions the user can take to meet their budgeting goals."

Reply:
resolved